src/Exporter.php

```diff
@@ -236,7 +236,7 @@ protected function recursiveExport(&$value, $indentation, $processed = null)
             }
 
             return "'" .
-            str_replace(array("\r\n", "\n\r", "\r"), array("\n", "\n", "\n"), $value) .
+            str_replace(array("\r\n", "\n\r", "\r", "\n"), array('<crlf>', '<lfcr>', '<lf>', '<ln>'), $value) .
```

Comment:
The Unicode `NEL` character (see https://en.wikipedia.org/wiki/Newline) might be useful here.